tests/suites/test_suite_psa_its.data

```diff
@@ -1,71 +1,74 @@
 Set/get/remove 0 bytes
```

Comment:
Commit structure: please do only one thing per commit. “One thing” can be multiple changes if they're closely related, for example fixing a bug and adding a non-regression test and adding a changelog entry (but it's ok to do those separately, too). This pull request does at least two things: change existing tests not to use uid=0, and add a test for uid=0. Please make those at least two separate commits.